logger.go

```diff
@@ -65,12 +67,31 @@ type logger struct {
 	Logger kitlog.Logger
 }
 
-func (l *logger) With(keyvals ...interface{}) Logger {
-	return &logger{
-		Logger: kitlog.With(l.Logger, keyvals...),
+func (l *logger) Log(keyVals ...interface{}) error {
+	return l.Logger.Log(keyVals...)
+}
+
+func (l *logger) LogWithCtx(ctx context.Context, keyVals ...interface{}) error {
```

Comment:
In Go when you add another parameter to the function you skip `With` word. E.g.: https://golang.org/pkg/bytes/#NewBufferString

Reply:
Do you want me to change this? I do not mind at this point.

Reply:
I'd like to change that yes <3

Reply:
I'm fine with doing that myself as well.